cloud_experts_tutorials/cloud-experts-rosa-hcp-activation-and-account-linking-tutorial.adoc

```diff
@@ -25,35 +25,44 @@ If you have received a private offer for the product, make sure to proceed accor
 == Prerequisites
 
 * Make sure to log in to the Red{nbsp}Hat account that you plan to associate with the AWS account where you have activated {hcp-title} in previous steps.
-* Only a single AWS account that will be used for service billing can be associated with a Red{nbsp}Hat account. Typically an organizational AWS account that has other AWS accounts, such as developer accounts, linked would be the one that is to be billed, rather than individual AWS end user accounts.
-* Red{nbsp}Hat accounts belonging to the same Red{nbsp}Hat organization will be linked with the same AWS account. Therefore, you can manage who has access to creating {hcp-title} clusters on the Red{nbsp}Hat organization account level.
+* The AWS account used for service billing can only be associated with a single Red{nbsp}Hat account. Typically a main AWS account that has other AWS linked accounts, such as developer accounts, would be the one that is to be billed, rather than individual AWS user accounts. In such cases, the main AWS account would be the one that is used to subscribe to ROSA and used for account linking and billing.
```

Comment:
```suggestion
* The AWS account used for service billing can only be associated with a single Red{nbsp}Hat account. Typically a main AWS account that has other AWS linked accounts, such as a developer account, rather than individual AWS user accounts is the AWS account used for service billing. The main AWS account is the one that is used to subscribe to ROSA and used for account linking and billing.
```